Classes/FrontEnd/RegistrationForm.php

```diff
@@ -771,13 +772,24 @@ public function populateListPaymentMethods(): array
             return [];
         }
 
-        $rows = \Tx_Oelib_Db::selectMultiple(
-            'uid, title',
-            'tx_seminars_payment_methods, tx_seminars_seminars_payment_methods_mm',
-            'tx_seminars_payment_methods.uid = tx_seminars_seminars_payment_methods_mm.uid_foreign ' .
-            'AND tx_seminars_seminars_payment_methods_mm.uid_local=' . $this->getSeminar()->getTopicOrSelfUid() .
-            \Tx_Oelib_Db::enableFields('tx_seminars_payment_methods')
-        );
+        /** @var \TYPO3\CMS\Core\Database\Query\QueryBuilder $qb */
+        $qb = GeneralUtility::makeInstance(ConnectionPool::class)
```

Comment:
Please add a local variable `$connectionPool` so we can have a `@var` annotation. (Then the other annotation is not needed.)

No abbreviations, please. `$qb` -> `$queryBuilder`